db/schema.rb

```diff
@@ -219,7 +218,7 @@
     t.integer "project_card_count"
     t.string "project_card_column_names"
     t.string "project_card_column_project_names"
-    t.string "project_card_project_urls"
+    t.string "project_card_column_project_urls"
```

Comment:
Do you know why this field name changed?

It seems like this would need to be a database migration, so I'm wondering if this is actually a glitch of some kind.

Reply:
Nvm... we discussed this related to Ahsun's PR, and it's fine.